common/app/routes/challenges/components/Classic-Modal.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { PropTypes } from 'react';
+import { Button, Modal } from 'react-bootstrap';
+import PureComponent from 'react-pure-render/component';
+import FontAwesome from 'react-fontawesome';
+
+import { randomCompliment } from '../../../utils/get-words';
+
+export default class ClassicModal extends PureComponent {
+
+  renderIcon() {
+    return (
+      <FontAwesome className='completion-icon text-primary'
+        name='check-circle'
+      />
+    );
+  }
+
+  renderSuccessMessage() {
+    return randomCompliment();
+  }
+
+  render() {
+    let {
+      closeChallengeModal,
+      keyBindings,
+      show,
+      submitChallenge
+    } = this.props;
+
+    return (
+      <Modal
+        animation={ false }
+        dialogClassName='success'
+        id='complete-courseware-dialog'
+        keyboard={ true }
+        onHide={ closeChallengeModal }
+        onKeyDown={ keyBindings }
+        onKeyUp={ keyBindings }
+        show={ show }
+        >
+        <Modal.Header className='challenge-list-header' closeButton={ true }>
+          <Modal.Title>{ this.renderSuccessMessage() }</Modal.Title>
```

Comment:
This still suffers from the same issue as before.

Maybe we can store the message in state and change it with every challenge transition?